sass/_ResponsiveTabs.scss

```diff
@@ -1,7 +1,8 @@
 .coveo-small-tabs {
 
-  .coveo-tab-dropdown-header {
+  #coveo-tab-dropdown-header {
```

Comment:
I need it to override a style specified by a sibling class. I preferred that over using a more specific selector. It would be easier to override for the users.